PhysicsTools/PatAlgos/python/recoLayer0/bTagging_cff.py

```diff
@@ -76,23 +76,24 @@
   , 'pfPositiveOnlyJetProbabilityBJetTags'                  : ['pfImpactParameterTagInfos']
   , 'pfNegativeOnlyJetBProbabilityBJetTags'                 : ['pfImpactParameterTagInfos']
   , 'pfNegativeOnlyJetProbabilityBJetTags'                  : ['pfImpactParameterTagInfos']
-  , 'pfTrackCountingHighPurBJetTags'                        : ['pfImpactParameterTagInfos']
+#  , 'pfTrackCountingHighPurBJetTags'                        : ['pfImpactParameterTagInfos']
```

Comment:
I would not comment out the TCHP algo in this file. Since all the producers still exist, I would leave the possibility of running it still available.